packages/now-cli/src/util/emoji.ts

```diff
@@ -0,0 +1,26 @@
+export function emoji(label: string): string | undefined {
+  switch (label) {
+    case 'notice':
+      return '📝';
+    case 'tip':
+      return '💡';
+    case 'warning':
+      return '❗️';
```

Comment:
The red makes it seem like an error. Perhaps:

```suggestion
      return '⚠️';
```

Reply:
Meh, nvm. It was the yellow emoji in the picker but I guess it has a plain fallback.